Core/HLE/sceMpeg.cpp

```diff
@@ -34,6 +34,7 @@
 #include "Core/Reporting.h"
 #include "GPU/GPUInterface.h"
 #include "GPU/GPUState.h"
+#include "CORE/HLE/sceKernelMemory.h"
```

Comment:
This needs to be changed to "Core/HLE/sceKernelMemory.h" . Some systems are case sensitive.